src/Query/Builder.php

```diff
@@ -923,6 +923,10 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
             }
         }
```

Comment:
Can't comment on the `if` itself, but I'm wondering if the condition should read `func_num_args() >= 3` - we wouldn't be removing the `$` from an operator if `$boolean` was defined as well 🤔

Reply:
It's not clear to me how `$boolean` is used, but I see it's passed on to the parent method below. I think that's right, and this suggests another lingering bug.

I'll defer to you guys whether to fix here (if so, add a regression test that passes a fourth parameter to `where()` (I suppose an explicit `'and'` string is fine) or report a new JIRA issue and address in a subsequent PR. A new issue probably makes more sense for visibility in the release notes.

Reply:
Fixed in https://github.com/GromNaN/laravel-mongodb/pull/20